src/app/space-wizard/common/logger.ts

```diff
@@ -59,23 +59,44 @@ export class LoggerFactory {
       if ( entry.info === true ) {
         method = 'info';
       }
+      let msg = `${origin} ${instance} ${entry.message || ''}` 
       console[ method ].apply(
         null,
         [ `%c${origin}%c ${instance} %c${entry.message || ''}`,
           me.styles.origin,
           me.styles.instance,
           me.styles.message ]
       );
+
+      if ( args && args.length === 1 ) {
+          if (typeof(args[0]) === 'function') {
+            args[0](msg);
+          }
+          else {
+            console.dir(args[0]);
+          }
+      }
+      if ( args && args.length > 1 ) {
+          if (typeof(args[0]) === 'function') {
+            args[0](msg);
+          }
+          else {
+            console.dir(args[0]);
+            if (typeof(args[1]) === 'function') {
+              args[1](msg);
+            }
+          }
+      }
     }
 
-    function loggerDelegate(options: string | ILogEntry) {
+    function loggerDelegate(options: string | ILogEntry,...args) {
       let entry = { message: '' };
       if ( typeof options === 'string' ) {
         entry.message = options || '';
       } else {
-        Object.assign(entry, options);
+        Object.assign(entry,options);
       }
-      addLogEntry(entry);
+      addLogEntry(entry,...args);
```

Comment:
pretty certain it is ... basically put the rest of the arguments in an array ... and second apply the array as an array instead of one argument as an array